security/pkg/stsservice/test/testdata/server-key.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
where does this come from? WE have lots of issues of certs expiring and have no indication of how to regen

Reply:
These certs will expire in 2030, and they are from https://github.com/istio/istio/blob/master/tests/integration/security/sds_ingress/util/test_certs.go.
I generate them following this guide https://istio.io/docs/tasks/traffic-management/ingress/secure-ingress-mount/#generate-server-certificate-and-private-key